### DISCUSSION TITLE
My language card not updated

### DISCUSSION THREAD

**Discussion by shariful10** [anuraghazra/github-readme-stats#3498] [Q&A] [answered]
### Describe the bug

![image](https://github.com/anuraghazra/github-readme-stats/assets/98546214/df14d70e-c526-46df-ab0f-38f8358f7660)

**Comment by qwerty541** [comment]
Hey, @shariful10! We're using caching to decrease load on public instance, default cache time is 21600 seconds (6 hours). Users also able to configure cache time to lower value after [deploying own instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) with and configuring `CACHE_SECONDS` environment variable. I see that you are using `https://denvercoder1-github-readme-stats.vercel.app`, so cache time depends from how it's owner configured it.

**Reply by shariful10** [reply]
how can I fix the issue? give me a solution, please.

**Reply by qwerty541** [reply, marked as answer]
I see several options for you:

- If you want your stats card to be update more often, the best solution for you is [deploying own instance](https://github.com/anuraghazra/github-readme-stats#on-vercel). As i previously said, you would be able to set `CACHE_SECONDS` environment variable with any value.
- Second option, using public deployment `github-readme-stats.vercel.app`, but you have to wait 6 hour before your card will be updated.
- Third option, continue using `https://denvercoder1-github-readme-stats.vercel.app`, with unknown cache time. It depends from how it's owner configured it and we can't know it.

**Reply by shariful10** [reply]
Thank you so much

**Reply by qwerty541** [reply]
You're welcome

**Reply by rickstaa** [reply]
@shariful10 In addition to your @qwerty541's points, the current language card has a few known bugs that might lead to inaccuracies in the language results for users with over 100 repositories (check out the details here: https://github.com/anuraghazra/github-readme-stats/issues/1852). I plan to address this by rolling out a [GitHub action](https://github.com/anuraghazra/github-readme-stats/issues/2179) in the first quarter of the upcoming year. Also, our language calculations are based on repository language results (more details at https://github.com/anuraghazra/github-readme-stats/issues/1801) due to GitHub's GraphQL API limitations. If you're keen on enhancing the language results, your support would be greatly appreciated at https://github.com/orgs/community/discussions/18230 🙏🏻. Thanks!
